recipes-wpe/wpeframework/wpeframework-rdkservices_git.bb

```diff
@@ -20,11 +19,11 @@ include include/monitor.inc
 
 # ----------------------------------------------------------------------------
 
-WPEFRAMEWORK_LOCATIONSYNC_URI ?= "http://jsonip.metrological.com/?maf=true"
+WPEFRAMEWORK_LOCATIONSYNC_URI ??= "http://jsonip.metrological.com/?maf=true"
```

Comment:
Should be groupped with the other configuration flags

Reply:
This file was not updated, since the current version of rdkservices plugins are not synced with the latest thunder. Restructured this now to use rdkservices-plugins-rdk.inc to get config details. But not tested